roles/tendrl-server/tasks/main.yml

```diff
@@ -28,6 +28,7 @@
 - include: tendrl-ui.yml
 - include: tendrl-monitoring-integration.yml
 - include: tendrl-notifier.yml
+- include: firewalld.yml
```

Comment:
This needs to moved up in the order. In a rare case where firewalld is not installed, it might create issues while installing tendrl-api etc as the /usr/lib/firewalld folder doesn't exists

Reply:
Sure, if that is the case, i will better move this to top in order

Reply:
I don't think that is required. Firewalld is expected to be available with base OS